src/share-element.js

```diff
@@ -0,0 +1,53 @@
+import {LitElement, html, css} from 'lit-element';
+
+class ShareElement extends LitElement {
+
+    static get properties() {
+        return {
+            locatorUrl : {type: String},
+            domain : {type: String},
+        }
+    }
+
+    constructor() {
+        super();
+        this.locatorUrl = encodeURI("https://chrome.google.com/webstore/detail/custom-elements-locator/eccplgjbdhhakefbjfibfhocbmjpkafc");
+    }
+
+    render() {
+        const tweetText = encodeURI(`I just found some custom-elements on ${this.domain} using locator!`);
+        return html`
+            <div class="share">
+                <img class="icon" src="./share.svg"/>
```

Comment:
```suggestion
                <img alt="" class="icon" src="./share.svg"/>
```

or `<title>` in the svg 🙂

Reply:
good point